Randomizer.SMZ3/Regions/SuperMetroid/NorfairLower/East.cs

```diff
@@ -37,14 +37,22 @@ class East : SMRegion, IReward {
 
         bool CanExit(Progression items) {
             return Logic switch {
-                Normal => items.CardNorfairL2 /*Bubble Mountain*/ ||
-                    items.Gravity && items.Wave /* Volcano Room and Blue Gate */ && (items.Grapple || items.SpaceJump /*Spikey Acid Snakes and Croc Escape*/),
-                _ => /*Vanilla LN Escape*/
-                    (items.Morph && (items.CardNorfairL2 /*Bubble Mountain*/ || (items.Missile || items.Super || items.Wave /* Blue Gate */) && 
-                                     (items.SpeedBooster || items.CanFly() || items.Grapple || items.HiJump && 
-                                     (items.CanSpringBallJump() || items.Ice) /*Frog Speedway or Croc Escape*/)) ||
-                    /*Reverse Amphitheater*/
-                     items.HasEnergyReserves(5)),
+                Normal => /* Intended LN Escape */
+                    items.Morph && (
+                        items.CardNorfairL2 /* Bubble Mountain */ ||
+                        items.Gravity && items.Wave /* Volcano Room, Blue Gate */ &&
+                            (items.Grapple || items.SpaceJump) /* Spikey Acid Snakes -> Croc Escape */
```

Comment:
Shouldn't there be a check for items.SpeedBooster here as well to account for exiting through frog speedway?

Reply:
Tar is correct. I will add a comment to make the reasoning apparent.